tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "ES2015",
+    "module": "commonjs",
```

Comment:
I noticed that in `tfjs`, there are two different `tsconfig` files; one which uses `ES2015` and ends up in `dist-es6`, and another which uses es5-compatible `commonjs` and ends up in `dist`. For our application, does it make sense to just use a single `tsconfig` with commonjs and writing into dist?

Reply:
makes sense. we can revisit later but this is good for now!